Tuist/ProjectDescriptionHelpers/Target+Initializing.swift

```diff
@@ -0,0 +1,46 @@
+import ProjectDescription
+
+extension Target {
+
+    public static func mainTarget(name: String) -> Target {
+        return Target(
+            name: name,
+            platform: .iOS,
+            product: .app,
+            bundleId: "co.nimblehq.\(name)",
+            infoPlist: .extendingDefault(with: [
+                "UILaunchStoryboardName": "LaunchScreen"
+            ]),
```

Comment:
I suggest using `.default` first. Because we will need to update again this parameter with a `path`. Let's focus on creating targets in this PR :D

Reply:
Updated in [7e24875](https://github.com/nimblehq/ios-templates/pull/164/commits/7e248759c063cee156452ddbf79904f78a17f727)